### PR TITLE
Add more documentation for the new command configuration options

### PR DESCRIPTION
**Pull request by pcarranza**

### CODE REVIEW

File: content/creating-commands.md [hunk 1]
```diff
@@ -35,13 +35,18 @@ A command can be configured the following way:
...
-- `auth_strategy`: defined the authorization strategy
+- `auth_strategy`: defines the authorization strategy
   - `any`: everyone will be allowed to run this command
   - `none`: no user will be allowed to run this command (default value,
     permissions have to be explicit and conscious)
   - `group`: use `allowed_groups` to control who has access to this command
 - `allowed_groups`: list of groups allowed to run this command
+- `channel_strategy`: defines the channel authorization strategy
+  - `any`: the command can be invoked from any channel
+  - `im_only`: the command can only be invoked in an IM conversation
```
Comment (omame) [inline content/creating-commands.md]: Isn't it DM? In any case, acronyms. 😡

Comment (pcarranza) [reply content/creating-commands.md]: I can change it.

Not sure about making it too large, but in any case, what's the specific proposal?

Comment (omame) [reply content/creating-commands.md]: >the command can only be invoked in an direct message conversation

or
>the command can only be invoked in an private conversation

Something more explicit than an acronym.

Comment (pcarranza) [reply content/creating-commands.md]: gotcha, now I understand what you were talking about.

Will do.

Comment (pcarranza) [reply content/creating-commands.md]: BTW, IM comes from the Slack API itself: https://github.com/nlopes/slack/blob/db68538b374a5f052bf6befea117ac74760f4e8e/websocket_dm.go#L4

**Review by omame — APPROVED** [review]

**Pull request merged**